config/application.php

```diff
@@ -95,6 +95,10 @@
     require_once $env_config;
 }
 
+if (isset($_SERVER['HTTP_X_FORWARDED_PROTO']) && $_SERVER['HTTP_X_FORWARDED_PROTO'] == 'https') {
```

Comment:
Might be useful to provide a comment referencing https://codex.wordpress.org/Function_Reference/is_ssl#Notes as well